cpp/inc/bond/core/reflection.h

```diff
@@ -200,6 +200,37 @@ const typename method_pointer<Service, Input, Result>::type
     MethodTemplate<Service, Input, Result, method_ptr, metadata_ptr>::method = method_ptr;
 
 
+/// @brief Method description in compile-time schema
+template <
+    typename Service,
+    typename Input,
+    typename Result,
+    const Metadata* metadata_ptr>
+    struct MethodTemplateNoFunc
```

Comment:
The name here isn't great. That appears to be because we're avoiding breaking back compat, by renaming `MethodTemplate`; however, only Comm uses `MethodTemplate`, so if we want to swap names, we can.

Or, possibly call this `MethodTemplateBase` and have `MethodTemplate` inherit from it?

Reply:
I think I'd rather pull this out into a separate bond::ext::grpc::reflection.h header. Sound OK?

Reply:
I like it even better.